File: drk-tool-proj/src/main/java/com/drk/tools/model/AgriQuantityLossChart.java
package com.drk.tools.model;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class AgriQuantityLossChart {

	private int id;
	private int commodityId;
//	private int phenophaseId;
	private int stressId;
	private float minBandValue;
	private float maxBandValue;
	private float minQuantityCorrectionPercent;
	private float maxQuantityCorrectionPercent;
	private String status;
	private Date createdAt;
	private Date updatedAt;

	@Override
	public String toString() {
		return id + ", " + commodityId + ", " /*+ phenophaseId + ","*/ + stressId + "," + minBandValue
				+ "," + maxBandValue + "," + minQuantityCorrectionPercent + "," + maxQuantityCorrectionPercent + ","
				+ "'" + status + "'," + createdAt + "," + updatedAt + " ";
	}

}

File: master-data-ui-proj/src/app/views/agri/models/AgriQuantityLossChart.ts
export class AgriQuantityLossChart{
    id : number;
    commodityId : number;
    commodity : string;
    stressId : number;
    stress : string;
    phenophaseId : number;
    phenophase : string;
    minPercentYieldCorrection : number;
    maxPercentYieldCorrection : number;
    avgSeverityBandMin : number;
    avgSeverityBandMax : number;
    // criteriaId : number;
    // criteria : string;
    status: string;
    minQuantityCorrectionPercent : number;
    maxQuantityCorrectionPercent : number;
    maxBandValue : number;
    minBandValue : number;
}